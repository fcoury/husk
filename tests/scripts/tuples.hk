let a = (1, 2, 3);
let b = (4, 5);

println!("Tuple a: {}", a);
println!("Tuple b: {}", b);
println!("Sum of first elements: {}", a.0 + b.0);
println!("Sum of second elements: {}", a.1 + b.1);
