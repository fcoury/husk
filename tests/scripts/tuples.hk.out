Tuple a: (1, 2, 3)
Tuple b: (4, 5)
Sum of first elements: 5
Sum of second elements: 7
